packages/babel-helper-remap-async-to-generator/src/index.js

```diff
@@ -145,6 +145,15 @@ function plainFunction(path: NodePath, callId: Object) {
     ]);
     declar._blockHoist = true;
 
+    if (path.parentPath.isExportDefaultDeclaration()) {
+      // change the path type so that replaceWith() does not wrap
+      // the identifier into an expressionStatement
+      path.type = "FunctionExpression";
+      path.parentPath.insertBefore(declar);
+      path.replaceWith(t.identifier(asyncFnId.name));
```

Comment:
Can you guys explain what the difference is between `export default x;` and `export { x as default};`. It looks exactly the same to me, but seems I'm missing something.